### PR TITLE
Update ingress send legal mail staff UI

### PR DESCRIPTION
**Pull request by dhirajshettymoj**
Update ingress for send legal mail staff UI based on the new templated and pointing to https://check-rule39-mail-<env>.prison.service.justice.gov.uk/

### CODE REVIEW

File: bin/record-build-info [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env node
```
Comment (psoleckimoj) [inline bin/record-build-info:1]: This isn't how the template app does this any more so depends what you want to do in that respect, or update at a later date.

**Review by psoleckimoj — APPROVED** [review]
LGTM!

File: bin/record-build-info (hunk 1 above)
Comment (dhirajshettymoj) [reply bin/record-build-info:1]: should I remove this bit? It is in the orginal send-legl-mail-to-prisons app so I copied it over. However am unsure as to why its needed

Comment (tpmcgowan) [reply bin/record-build-info:1]: I couldn't find any reference to it in the code so I reckon you could get rid of it (and the corresponding line added to `package.json`).

It was removed from the template app here (https://github.com/ministryofjustice/hmpps-template-typescript/pull/190) - I'm assuming this repo was cloned from the template after this.

**Review by tpmcgowan — APPROVED** [review]
LGTM

**Pull request merged**